tests/src/test/scala/tests/SerializationSuite.scala

```diff
@@ -113,33 +112,12 @@ class SerializationSuite extends AnyFunSuite {
     val bsp4jDiagnostic1 = bsp4jValue.getDiagnostics.get(0)
     assert(bsp4jDiagnostic1.getMessage == bsp4sValue.diagnostics.head.message)
     assert(
-      bsp4jDiagnostic1.getSeverity.getValue == bsp4sValue.diagnostics.head.severity.get.id
+      bsp4jDiagnostic1.getSeverity.getValue == bsp4sValue.diagnostics.head.severity.get.value
     )
     assert(bsp4sValueDecoded.diagnostics.head.data == diagnostic1.data)
     assert(bsp4sValueDecoded == bsp4sValue)
   }
 
-  test("BuildTargetCapabilities - backward compatible canDebug") {
```

Comment:
what happened to this?

Reply:
BuildTargetCapabilities in bsp4s had a `canDebug: Boolean = false` field. It was the only field with a default value, so I didn't write a generator for default values. `canDebug` became an obligatory field in the spec some time ago

Reply:
I have an idea now: we can make all fields of `BuildTargetCapabilities` optional in the spec, as null amounts to the same as false in this case.

Reply:
sounds good to me